CONTRIBUTING.md

```diff
@@ -1,16 +1,39 @@
 # Contributing
 
-Thank you so much for taking the time to contribute to Mapeo development!
-mapeo-mobile is a mobile app written in react-native that currently runs on
-Android. For most development you will need the Android SDK installed on your
-computer, but it is possible to contribute to UI Component development using the
-storybook server.
+Thank you so much for taking the time to contribute to Mapeo development! Mapeo-mobile is a mobile app written in react-native that currently runs on Android mobile devices. 
 
-## Initial Install
+## Setting Up the Development Environment
 
-In order to start developing you will need git and node >=v10 installed on your
-computer (node v14 does not seem to be working at this time, you can use [nvm](https://github.com/nvm-sh/nvm) to rollback
-your node version). For many development tasks you will also need the Android SDK installed.
+To get started with Mapeo-mobile development, you need some basic software installed on your computer. Please ensure you have correct version of required software for pain-free development experience. 
+
+### Prerequisites
+
+* A OSX, gnu/Linux, Windows OS (64bit) computer with plenty of disk space and multi-core CPU for faster builds.  
+* Node 12 (<=12.16.3)
+* NPM (Node Package Manager) or Yarn 
```

Comment:
wondering if it's worth mentioning the versions of npm and yarn here. For npm, I know that Node@12 comes with npm@6 by default (latest stable is 8 – not even sure if it's compatible with Node@12). Haven't tried using yarn in this repo but my guess is that only yarn 1 will work

Reply:
Will need to check with @gmaclennan Last year we did transition from npm to yarn workspaces.